apollo-router/src/plugins/telemetry/otlp/mod.rs

```diff
@@ -47,6 +48,33 @@ impl Exporter {
             Exporter::Http(exporter) => Ok(exporter.clone().unwrap_or_default().exporter()?.into()),
         }
     }
+
+    pub fn metrics_exporter(
+        &self,
+    ) -> Result<opentelemetry_otlp::MetricsExporter, ConfigurationError> {
+        match &self {
+            #[cfg(feature = "otlp-grpc")]
+            Exporter::Grpc(_exporter) => {
+                // let grpc_exporter = exporter.clone().unwrap_or_default();
+                // let tonic_exporter = grpc_exporter.exporter()?;
+                todo!("TonicConfig struct is not already public in opentelemetry_otlp");
+                // opentelemetry_otlp::MetricsExporter::new(
+                //     *tonic_exporter.export_config(),
+                //     tonic_config,
+                //     export_selector,
+                // );
+            }
+            #[cfg(feature = "otlp-http")]
+            Exporter::Http(_exporter) => {
+                todo!("TonicConfig struct is not already public in opentelemetry_otlp");
+                // opentelemetry_otlp::MetricsExporter::new(
+                //     *tonic_exporter.export_config(),
+                //     tonic_config,
+                //     export_selector,
+                // );
+            }
+        }
+    }
```

Comment:
Does this mean we are waiting for changes to the opentelemetry crate to be able to do this?